pkg/k8s_tap_service_test.go

```diff
@@ -0,0 +1,37 @@
+package kubeturbo
+
+import (
+	"k8s.io/client-go/rest"
+	"testing"
+)
+
+func TestParseK8sTAPServiceSpec(t *testing.T) {
+	targetName := "target-foo"
+	configPath := "../test/config/turbo-config"
+
+	kubeConfig := &rest.Config{Host: targetName}
+
+	got, err := ParseK8sTAPServiceSpec(configPath, kubeConfig)
+
+	if err != nil {
+		t.Errorf("Error while parsing the spec fiel %s: %v", configPath, err)
```

Comment:
typo ```t.Errorf("Error while parsing the spec field %s: %v", configPath, err)```
also, you should return here, right?

Reply:
Fixed the typo. t.Errorf will fail the test so no return needed. Thanks, Dongyi!!